App/Redux/FlowReducer.ts

```diff
@@ -0,0 +1,45 @@
+/// <reference path="../../typings/tsd.d.ts" />
+import {AuthStateEnum, PageVisibilityEnum, IErrorState} from './FlowActions';
+
+export interface IControlState{
+  authState: AuthStateEnum;
+  pageState: PageVisibilityEnum;
+  error: IErrorState;
```

Comment:
IErrorState is an action type that you defined in FlowActions.ts. We shouldn't use it in the store.